### ISSUE TITLE
Update website

### ISSUE THREAD

**Issue by dragos** [mucommander/mucommander#12]
The official site is misleading: http://www.mucommander.com/

All links are dead (forum, bug reports, source, etc.). There is no easy way to find out that the project is still alive, and for some reason the official site is still top-hit in Google. Please update it and add a link to this GitHub repository.

**Comment by catalin-hritcu** [comment]
I think this would indeed be great.

Problem is that at the moment documentation is spread all over, including lots of dead links. Has anyone figured out how to build the github version for instance? I run across a half-dead link:
http://trac.mucommander.com/wiki/QuickStartGuide saying 
```
Error

TracError: The Trac Environment needs to be upgraded.

Run "trac-admin /var/trac/mucommander upgrade"
```

**Comment by catalin-hritcu** [comment]
Finally figured out how to build this despite the outdated compile.txt (which says one should use ant, when in fact this needs gradle):
```
git clone git@github.com:mucommander/mucommander.git
git submodule update --init --recursive
gradle build
gradle run
```
Any chance of adding (something like) this to README.md ... and removing obsolete documentation and broken links?

**Comment by burner1024** [comment]
Summoning both @ahadas and @nrinaudo here. Please respond. If you aren't not going to continue supporting, let people know that you disown the project.

**Comment by ahadas** [comment]
Guys, sorry for the late response.
We (Maxence, Nicolas and I) are discussing the future of mucommander these days. We are considering few alternatives and hopefully we will decide soon and publish our decision.
I agree that it is a pity that the website is broken as @dragos  wrote and the documentation is also not up-to-date as @catalin-hritcu  wrote, especially since I see that there is an interest in the project - it is still being forked and I still get questions about the project as in this thread.
However, I believe that in this stage it is important to decide where we are heading to both technically and in regards to the community.

**Comment by begray** [comment]
Guys, do you have any news on this?

muCommander - is a lifesaver for me personally. I've tried a lot of other file managers on MacOS, but any other doesn't seem right. 

Current versions on website and in this github repository have major issues with support of latest Java and MacOS versions. Some of them we solved in forks. The problem is there is a number of forks right now and no clue which one to choose from them.

**Comment by Meilenstein** [comment]
Any news?

**Comment by ahadas** [comment]
Hi all,
Few updates:
1. We agreed that the maintainership model in the project will change. I will update about this soon.
2. The website will be fixed and will move to github pages.
3. My time is limited these days so the above was postponed. Hopefully within few weeks I will be able to move the website, release the next stable release and then continue to migrate other resources.
Stay tuned!

**Comment by ahadas** [comment]
Guys,
I would like to update you that the new website is in progress.
It is not much different than the previous one (which is currently pointed by www.mucommander.com), and some of the links are "under-construction".
Later on, it should probably be replaced with something that is more tailored to github-pages.
I would love to get feedback on the new website: http://mucommander.github.io
Note that the biggest changes are:
1. the news section is generated from twitter
2. the nightly builds were removed - need to think how to share nightly build.
Your comments are welcome.

**Comment by aadrian** [comment]
> I would love to get feedback on the new website: http://mucommander.github.io
> 1. the news section is generated from twitter

Thank you for the updates.
 - News show nothing right now.
 - There are still links in that page to Trac that are dead, and also other dead links (showing to 404 pages).

> 2. the nightly builds were removed - need to think how to share nightly build.

Since the last release is so long time ago, maybe it would be time to some "alpha" or "beta" release for now a solution?

**Comment by ahadas** [comment]
@aadrian 
1. Can't you see the item "The website is being moved to GitHub pages!" in the news?
2. Yeah, there is one link to the Trac that I missed, did you see more than one?
3. Well, the problem is with the packaging. Two things should be done before we will be able to release a new version:
* to fix the packaging so it would work with gradle as it worked in the past
* to verify that our up-to-date source code can run with Java 6 on mac os x (otherwise we should probably bundle the jre in our dmg file)
And when we do it, I think we can then release the next version - without any 'alpha', 'beta' or 'release candidate' version. It seems that current master branch is pretty stable

**Comment by aadrian** [comment]
> 1. Can't you see the item "The website is being moved to GitHub pages!" in the news?

That's true for the link inside the page, but not for the news anchor in the top menu: that menu item has no effect.

IMHO for OSX there's no need to make it java 6 compatible, nor bundle a JRE: most installations of mucommander on OSX I've seen, were with brew, and there's brew cask for an recent Java version (so a dependency can be made), or for the other non-brew users (and for most other Java based applications), the users will install the Oracle JDK manually.

**Comment by Pasqualle** [comment]
![web-desktop](https://user-images.githubusercontent.com/104823/107132868-4508c680-68e3-11eb-85b8-5b70787a8add.png)
![web-mobile](https://user-images.githubusercontent.com/104823/107132871-489c4d80-68e3-11eb-9f0d-8d109fa54b8f.png)
![web-tablet](https://user-images.githubusercontent.com/104823/107132873-4934e400-68e3-11eb-81ae-1164608ace0a.png)

I have prepared a website update.

**Comment by ahadas** [comment]
@Pasqualle it looks great!
I will look at the PR soon

**Comment by ahadas** [comment]
@Pasqualle I've merged the PR, thank you for this contribution!
Not sure about Discord but let's discuss it [separately](https://github.com/mucommander/mucommander.github.io/pull/3)
With that, and considering other links and instructions on how to build muCommander were updated already, I think it's time to close this issue
Thanks everyone who provided feedback on that
